packages/flame/src/Checkbox/README.md

```diff
@@ -131,13 +121,42 @@ const MyComponent = () => (
     <Checkbox
       id="feature-id"
       name="feature"
-      label={labelId => <div id={labelId}>Custom Label</div>}
-      description={descriptionId => (
-        <SomeCustomComponent id={descriptionId}>Custom description</SomeCustomComponent>
-      )}
+      label={<div>Custom Label</div>}
+      description={<SomeCustomComponent>Custom description</SomeCustomComponent>}
     />
   </div>
 );
 
 export default MyComponent;
 ```
+
+#### `<BaseCheckbox>`
+
+The primitive element of Checkbox. No labels or descriptions.
+
+This component has the exact same props as the native HTML checkbox.
+
+#### `<CheckboxLabel>`
+
+Specially styled label to be used in conjunction with the BaseCheckbox component. Use this component to properly align the description.
+
+| Prop            | Type                         | Default   | Description                                                    |
+| --------------- | ---------------------------- | --------- | -------------------------------------------------------------- |
+| `description`   | `string` or `child function` | undefined | Description's text                                             |
+| `html property` | `string`                     | undefined | Any extra properties passed will be added to the `<label>` tag |
```

Comment:
Not sure I understand `html property`

Reply:
Any valid HTML properties such as `id`, `for`, etc.

Reply:
OK, I would add an example then, because it's not very clear.